test/Microsoft.Extensions.Internal.Test/project.json

```diff
@@ -3,49 +3,25 @@
     "embed": {
       "include": "../../src/*.Sources/**/*.resx"
     },
-    "warningsAsErrors": true
+    "warningsAsErrors": true,
+    "compile": [
+      "../../src/Microsoft.Extensions.ClosedGenericMatcher.Sources/**/*.cs",
```

Comment:
Ehhhh........ can this continue to use packages? I'm worried that by not using the packages we won't detect problems in the packages themselves and the build will fail waaay later.